builder/assignmentEnvImageBuilder.go

```diff
@@ -88,7 +100,10 @@ func (assgnEnv *assignmentEnvironment) verifyAndWriteInstructions() error {
 	return nil
 }
 
-func (assgnEnv *assignmentEnvironment) verifyLanguage() error {
+// verifyLanguage searches the docker image for the given language of docker hub.
```

Comment:
previously used "language image" and now using "docker image for the given language".
It is better to stick to one usage. I prefer "docker image for  ...".

Reply:
Changed